mulighetsrommet-api/src/main/resources/application-local.yaml

```diff
@@ -57,3 +57,11 @@ app:
   poaoTilgang:
     url: http://localhost:8090/poao-tilgang
     scope: default
+
+  amtEnhetsregister:
+    url: http://localhost:8090
```

Comment:
Er disse midlertidige?

Reply:
Stryk det, ser at det gjelder for de andre klient-configene også.

Siden det blir såpass mange http clients etter hvert så kunne det kanskje vært en fordel å legge på et lag i url'ene til mock-serveren i tilfelle noen av disse tjenestene har samme endepunkter?
Altså tilsvarende det som er gjort med poaoTilgang-klienten. Kan i tilfelle gjøres som en separat opprydningsoppgave :)